src/ray/core_worker/transport/direct_task_transport.h

```diff
@@ -339,6 +352,18 @@ class CoreWorkerDirectTaskSubmitter {
       // If any worker has more than one task in flight, then that task can be stolen.
       return total_tasks_in_flight > active_workers.size();
     }
+
+    // Get the current backlog size for this scheduling key
+    inline int64_t BacklogSize() const {
+      if (task_queue.size() < pending_lease_requests.size()) {
+        // During work stealing we may have more pending lease requests than the number of
```

Comment:
@stephanie-wang do we have a timeline to discuss whether to remove work stealing?

Reply:
Ah not yet, I'll reach out to Gabriele.